components/core/src/reset.js

```diff
@@ -2,15 +2,18 @@
 
 import { jsx, css } from '@emotion/core';
 
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+/* Note: The following styles were not included as they apply to the document specifically...
+html {
+	line-height: 1.15;
+	-webkit-text-size-adjust: 100%;
+}
+body {
+	margin: 0;
+} 
+*/
+// TODO: consider moving to <Global />, but note will apply to document (outside of <GEL /> wrapper)
```

Comment:
This would break things what AEM wanted with multiple React apps in the same page

Reply:
The TODO comment was referring to just moving the two styles removed from the reset up into `<Global />`. Just these ones...

```
html {
	line-height: 1.15;
	-webkit-text-size-adjust: 100%;
}
body {
	margin: 0;
} 
```

They do nothing when rendered inside our wrapper. It would attempt to at least set the document styling to a reliable starting point. The `webkit-text-size-adjust` is important, margin reset too I guess. But yes moving these to Global space could possibly cause issues for existing pages, only minimal 'issues' though... what do you reckon @dominikwilkowski ?

Reply:
I would not touch things outside of your system. You don't know what the conditions are...